test_main.py

```diff
@@ -0,0 +1,36 @@
+"""
+This is the test !!!
+"""
+import unittest
+from app import get_guess_computer, get_guess_Player, mainFunction
+from unittest.mock import patch
+import io
+import random
+
+class TestMainCode(unittest.TestCase):
+    def test_get_computer(self):
+        with patch('sys.stdout', new= io.StringIO()) as fake_out:
+            result = get_guess_computer()
+            self.assertIsInstance(result, int)
+            self.assertGreaterEqual(result, 1)
+            self.assertLessEqual(result, 100)
+            self.assertEqual(fake_out.getvalue(), '')
+
+    def test_get_player(self):
+        with patch('sys.stdout', new= io.StringIO()) as fake_out:
```

Comment:
for `get_guess_player` I would not mock `stdout`. I don't think you should tests that this function prints a number (in fact, I don't think this function should do _any_ printing, just returning the number same for `test_get_computer`)

what you may need to do, is mock `input` because this function asks for the number from the input, and we want to test that it gives the same number as an `int`

For example, here we are forcing it to 22
`@patch('builtins.input', return_value='22')`

Reply:
ok